templates/bbb-web.properties

```diff
@@ -18,14 +18,16 @@ defaultWelcomeMessageFooter={{ bbb_dialin_footer }}
 {% else %}
 defaultWelcomeMessageFooter={{ bbb_default_welcome_message_footer }}
 {% endif %}
+defaultMaxUsers={{ bbb_default_max_users }}
+maxNumPages={{ bbb_max_num_pages }}
+maxFileSizeUpload={{ bbb_max_file_size_upload }}
```

Comment:
This entries already exist below.

Reply:
Not yet @Davka 
In the current master branch there are two times "defaultMaxUsers":
https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/bbb/2.3/templates/bbb-web.properties#L22
https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/bbb/2.3/templates/bbb-web.properties#L30

Reply:
I created a PR for this #132